src/ekf_state_est/src/adapter_ekf_state_est/adapter.cpp

```diff
@@ -5,6 +5,6 @@ Adapter::Adapter(std::shared_ptr<SENode> se_node) : node_(se_node) {
   RCLCPP_INFO(this->node_->get_logger(), "Adapter created");
 }
 
-void Adapter::imu_subscription_callback(const sensor_msgs::msg::Imu& msg) {
+void Adapter::imu_subscription_callback(const sensor_msgs::msg::Imu& msg) const {
```

Comment:
Sure it is a suggestion, but it is misleading to call this function const wouldn't you say?

Reply:
Sim, eu depois alterei isso nos outros, mas esqueci-me de alterar aqui. Já resolvo

Reply:
Até porque não podemos definir a callback const